crypto/kzg/src/trusted_setup.rs

```diff
@@ -30,6 +36,20 @@ pub struct TrustedSetup {
     g2_points: Vec<G2Point>,
 }
 
+impl Default for TrustedSetup {
+    fn default() -> Self {
+        if let Ok(trusted_setup) = serde_json::from_slice(get_trusted_setup().as_slice()) {
+            trusted_setup
+        } else {
+            Self {
+                g1_monomial_points: Default::default(),
+                g1_points: Default::default(),
+                g2_points: Default::default(),
+            }
```

Comment:
This default impl feels a little sketchy, as I imagine the inner `default` impls here use some junk values?

I think we could remove the need for this by making `client::Config::trusted_setup` either:

- a `Vec<u8>` which is parsed on node startup, or
- an `Option<TrustedSetup>`, which we error out on at startup if it is `None`

Reply:
The only pace the default `TrustedSetup` is used is in the default `client::Config` impl. I've moved the trusted setup parsing into the `Config` default impl, which will error out if the trusted_setup file isnt found. You think that should be ok? We could alternatively change  `client::Config::trusted_setup` to optional or a Vec<u8> as you said

Reply:
yeah that seems good, I would rather have the expect